DeepSpeech.ipynb

```diff
@@ -1371,9 +1376,9 @@
     "            'n_hidden_6': n_hidden_6, \\\n",
     "            'n_cell_dim': n_cell_dim, \\\n",
     "            'n_character': n_character, \\\n",
-    "            'total_batches_train': ted_lium.train.total_batches, \\\n",
-    "            'total_batches_validation': ted_lium.dev.total_batches, \\\n",
-    "            'total_batches_test': ted_lium.test.total_batches \\\n",
+    "            'total_batches_train': run_data_set.train.total_batches, \\\n",
+    "            'total_batches_validation': run_data_set.dev.total_batches, \\\n",
+    "            'total_batches_test': run_data_set.test.total_batches \\\n",
```

Comment:
As we are now switching data sets, we should log which data set the data is from. Something like
```python
...
        'data_set': { \
            'name': os.environ.get('ds_importer', 'ted_lium')
        }, \
...
```
Also, I'd guess [doyouunderstand.me](http://doyouunderstand.me) should now display the data set's name too.